src/test/cli/osdmaptool/crush.t

```diff
@@ -6,7 +6,7 @@
   osdmaptool: exported crush map to oc
   $ osdmaptool --import-crush oc myosdmap
   osdmaptool: osdmap file 'myosdmap'
-  osdmaptool: imported 497 byte crush map from oc
+  osdmaptool: imported 499 byte crush map from oc
```

Comment:
Hmm, 2*`sizeof(__u8)` maybe?

Reply:
It actually needs to be 505 now that the tunables are __u32, but I'm not sure what you mean.  There's no support here for sizeof.

Reply:
Sorry for confusion, I just wanted to point this might be the leftover from the 2 instances of `__u8` in crush map.

Reply:
Ah, yeah.